fastparquet/writer.py

```diff
@@ -814,15 +815,27 @@ def write(filename, data, row_group_offsets=50000000,
           mkdirs=default_mkdirs, has_nulls=True, write_index=None,
           partition_on=[], fixed_text=None, append=False,
           object_encoding='infer', times='int64'):
-    """ Write Pandas DataFrame to filename as Parquet Format
+    """ Write Pandas DataFrame to filename as Parquet Format.
+
+        3 writing modes can be identified:
```

Comment:
Please relegate this block to the `append` parameter; potentially take it out completely and put it in the docs. The reason I say this, is that the matter is hard to understand, and by trying to fit it here, it becomes rather terse. In the docs, you could add examples to show what overwrite and append modes are supposed to do.